### PR TITLE
feat: user config SSG, arrays

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

adds support for `array` types in user config static schema generator

## Why this way

to support `array` types :)

### CODE REVIEW

File: internal/schemautil/user_config/handle.go [hunk 1]
```diff
@@ -34,9 +34,64 @@ func handleObjectProperty(n string, p map[string]interface{}, t string) *jen.Sta
...
+
 // handleArrayProperty is a function that converts an array type property to a Terraform schema.
 func handleArrayProperty(n string, p map[string]interface{}, t string) *jen.Statement {
-	return jen.Values(convertPropertyToSchema(n, p, t))
+	ia, ok := p["items"].(map[string]interface{})
+	if !ok {
+		return nil
+	}
+
+	// TODO: Handle this case separately.
```
Comment (Serpentiel) [inline internal/schemautil/user_config/handle.go:62]: will be done in a separate PR

File: internal/schemautil/user_config/handle.go [hunk 2]
```diff
@@ -34,9 +34,64 @@ func handleObjectProperty(n string, p map[string]interface{}, t string) *jen.Sta
...
+	if _, ok := ia["one_of"]; ok {
+		return nil
+	}
+
+	// TODO: Handle this case separately. It accepts both object and string at the same time.
```
Comment (Serpentiel) [inline internal/schemautil/user_config/handle.go:67]: will be done in a separate PR

**Review by ivan-savciuc — APPROVED** [review]
Great job
